service/src/main/kotlin/io/provenance/explorer/domain/models/explorer/ExplorerModels.kt

```diff
@@ -28,3 +30,126 @@ data class GasStatistics(
     val maxGasPrice: Int,
     val averageGasPrice: BigDecimal
 )
+
+data class Params(
+    val cosmos: CosmosParams,
+    val prov: ProvParams,
+)
+
+data class CosmosParams(
+    val authParams: AuthParams,
+    val bankParams: BankParams,
+    val distParams: DistParams,
+    val govParams: GovParams,
+    val mint: MintParams,
+    val slashingParams: SlashingParams,
+    val stakingParams: StakingParams,
+    val ibc: IBCParams,
+)
```

Comment:
~~Update~~ Drop `params` from the field names to match up with the [query-commands](https://docs.provenance.io/blockchain/using-provenance/query-command) in the docs

auth
bank
distribution
gov
mint
slashing
staking
ibc